### PR TITLE
Tickets/dm36977: Added MTAlignment, MTAirCompressor and GenericCamera reports

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

File: QueryEfd.py [hunk 1]
```diff
@@ -239,33 +239,39 @@ def verify_summary_state(
...
             raise TypeError("Expected State must be an integer.")
         csc, index = self._split_indexed_csc(csc_str)
-        # Auto-Enabled CSCs automatically go into the Enabled State.
-        # As such, grab the 3 most recent samples, in order to properly
-        # test the SummaryState.
-        if auto_enable:
-            num = 3
+        # Auto-Enabled CSCs automatically go into the Enabled State,
+        # MTAirCompressors automatically go into the Disabled State,
+        # remaining CSCs go through States one-by-one.
```
Comment (mareuter) [inline QueryEfd.py:244]: change one-by-one to manually. The CSCs that use the auto state transitioning mechanism also go through each state.

File: QueryEfd.py [hunk 2]
```diff
@@ -239,33 +239,39 @@ def verify_summary_state(
...
             [
                 "private_sndStamp",
                 "summaryState",
             ],
-            num,
+            3,
             index,
         )
+        # Which row of the DataFrame is needed depends on the expected_state.
+        if row_index == 2 and expected_state == 5:
```
Comment (mareuter) [inline QueryEfd.py:264]: I see three statements here but the code triggered by each conditional is the same, so why do we need the conditionals at all?

Comment (rbovill) [reply QueryEfd.py]: Fixed and simplified.

**Review by mareuter — APPROVED** [review]
Looks fine to me. Thanks for the changes.

**Pull request merged**